packages/libs/lambda-at-edge/package.json

```diff
@@ -62,6 +62,7 @@
   },
   "dependencies": {
     "@aws-sdk/client-s3": "1.0.0-rc.3",
+    "@aws-sdk/client-sqs": "1.0.0-rc.3",
```

Comment:
I believe https://www.npmjs.com/package/@aws-sdk/client-s3/v/3.13.1 is the latest for both, maybe update this and client-s3 as well (I still need to get the dependency updates working...)

Reply:
👍